CSS/about-myself.css

```diff
@@ -0,0 +1,137 @@
+.about-myself {
+  background-color: #fff;
+  width: 100%;
+  height: 1200px;
+  padding: 0 24px;
+  border-top-right-radius: 100px;
+  margin-top: 60px;
+}
+
+.aboutmyself-title {
+  margin-left: 0;
+  margin-top: 114px;
```

Comment:
- [ ] kindly reduce  the margin-top so that it will reduce the space between cards and about section so that the design 
will match with the Figma design shown bellow \
## your design
![image](https://user-images.githubusercontent.com/95297251/185512306-2df0b301-6085-48da-9348-3e52136bc092.png)
 ## Figma design
![image](https://user-images.githubusercontent.com/95297251/185512534-14b3f2d5-cd3e-4416-9a7a-40dfe5a87071.png)

Reply:
but it is 114px as its shown in Figma as well, should I lower it more?.